data/tribes/initialization/atlanteans/units.lua

```diff
@@ -229,228 +326,466 @@ tribes:new_tribe {
+                  pgettext("ware", "Granite is a basic building material."),
+                  -- TRANSLATORS: Helptext for an atlantean ware: Granite, part 2
+                  pgettext("atlanteans_ware", "The Atlanteans produce granite blocks in quarries and crystal mines.")
+               }
+            }
          },
          {
             name = "log",
-            preciousness = 14
+            preciousness = 14,
+            helptexts = {
+               purpose = {
+                  -- TRANSLATORS: Helptext for an atlantean ware: Log, part 1
+                  pgettext("ware", "Logs are an important basic building material. They are produced by felling trees."),
+                  -- TRANSLATORS: Helptext for an atlantean ware: Log, part 2
+                  pgettext("atlanteans_ware", "Atlanteans use logs also as the base for planks, which are used in nearly every building. Besides the sawmill, the charcoal kiln, the toolsmithy and the smokery also need logs for their work.")
+               }
+            }
          },
          {
             name = "planks",
             default_target_quantity = 40,
-            preciousness = 10
+            preciousness = 10,
+            helptexts = {
+               purpose = {
+                  -- TRANSLATORS: Helptext for an atlantean ware: Planks, part 1
+                  pgettext("ware", "Planks are an important building material."),
+                  -- TRANSLATORS: Helptext for an atlantean ware: Planks, part 2
+                  pgettext("atlanteans_ware", "They are produced out of logs by the sawmill."),
+                  -- TRANSLATORS: Helptext for an atlantean ware: Planks, part 3
+                  pgettext("atlanteans_ware", "The weapon smithy and the shipyard also use planks to produce the different tridents and mighty ships.")
+               }
+            }
          },
          {
             name = "spider_silk",
             default_target_quantity = 10,
-            preciousness = 11
+            preciousness = 11,
+            helptexts = {
+               -- TRANSLATORS: Helptext for an atlantean ware: Spider Silk
+               purpose = pgettext("atlanteans_ware", "Spider silk is produced by spiders, which are bred by spider farms. It is processed into spidercloth in a weaving mill.")
+            }
          },
          {
             name = "spidercloth",
             default_target_quantity = 20,
-            preciousness = 7
+            preciousness = 7,
+            helptexts = {
+               -- TRANSLATORS: Helptext for an atlantean ware: Spidercloth
+               purpose = pgettext("atlanteans_ware", "Spidercloth is made out of spider silk in a weaving mill. It is used in the toolsmithy and the shipyard. Also some higher developed buildings need spidercloth for their construction.")
+            }
          },
       },
       {
-         -- Food
+         -- [[ Food ]]
```

Comment:
what is the reason for having these brackets here and not having them for the other categories?